### DISCUSSION TITLE
Welcome to Stormwater-Management-Model Discussions!

### DISCUSSION THREAD

**Discussion by michaeltryby** [USEPA/Stormwater-Management-Model#119] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dickinsonre** [comment]
Nice thread Michael,

**Comment by MitchHeineman** [comment]
Mitch Heineman here - I first used SWMM ca. 1988, and have used it nearly daily since 1993 as an engineer working on collection systems planning at CDM Smith in Boston. I've made many suggestions for code revisions over the years, have written a few bits of proposed code (e.g., what became the [EVENTS] feature), and have prototyped other proposed improvements (e.g., implementation of the Hargreaves evapotranspiration method in a spreadsheet). I've participated in the SWMM-USERS (a.k.a. OpenSWMM) discussion board since the mid-1990s. I've run many training classes for SWMM, and am the principal developer of www.dynsystem.com/netstorm, which provides pre- and post-processing support for SWMM.

**Reply by michaeltryby** [reply]
Welcome @MitchHeineman! Great to have you here

**Comment by cbuahin** [comment]
Welcome @MitchHeineman. As @michaeltryby mentioned on the other thread #127 , we are working on organizing ourselves so that we able to consider suggestions and code in a transparent, thoughtful, and manageable way that maintains the quality of the software we have all come to love and rely on. We are looking forward to your inputs.
